packages/plugin-mercury/src/mercury.js

```diff
@@ -300,6 +300,8 @@ const Mercury = SparkPlugin.extend({
       }), Promise.resolve())
       .then(() => {
         this._emit(`event`, event.data);
+        const [namespace] = data.eventType.split(`.`);
```

Comment:
I had to execute this out in node to figure out what `namespace` would be. Is there a reason to do this instead of `.split('.')[0]`?